### PR TITLE
Add new window icon

### LINKED ISSUE #456: Add new tab/window icon
Icon requested by @Alondite00 

screen shot 2016-04-04 at 3 08 48 pm

### PR DESCRIPTION
**Pull request by interactivellama**
Fixes #456

**Adds:**
screen shot 2016-04-04 at 3 08 48 pm

**However, this is often used as a share icon.**
screen shot 2016-04-04 at 3 40 44 pm

**I would recommend:**
screen shot 2016-04-04 at 3 40 32 pm

### CODE REVIEW

**Comment by interactivellama** [comment]
Moving forward with the "share icon" @Alondite00 If we decide to change this to the more square arrow, we can always just update the CSS.

**Comment by interactivellama** [comment]
@futuremint Updated.

File: less/icons/icons-png-fallback.css [hunk 1]
```diff
@@ -222,6 +222,8 @@
 
 .fuelux-icon-multimail, .glyphicon-multimail { background-image: url('png/multimail.png'); background-repeat: no-repeat; .fuelux-icon; } 
 
+.fuelux-icon-new-window, .glyphicon-new-window { background-image: url('png/new-window.png'); background-repeat: no-repeat; .fuelux-icon; }
```
Comment (futuremint) [inline less/icons/icons-png-fallback.css]: I don't see `png/new-window.png` in this PR anywhere... is it supposed to be here?

Comment (interactivellama) [inline less/icons/icons-png-fallback.css]: These appear to be gitignored. PNG usage is not prescribed and is a side effect of Grunt Icon believe. We probably should change the setup of Grunt Icon to not produce these.

https://github.com/ExactTarget/fuelux-mctheme/blob/master/.gitignore#L20

**Pull request merged**